setup.py

```diff
@@ -16,8 +16,8 @@
     name="spidermon",
     version="1.14.0",
     url="https://github.com/scrapinghub/spidermon",
-    author="Scrapinghub",
-    author_email="info@scrapinghub.com",
+    author="Zyte",
+    author_email="info@zyte.com",
```

Comment:
This may be a better choice.

```suggestion
    author_email="opensource@zyte.com",
```